sklearn/linear_model/logistic.py

```diff
@@ -999,7 +999,7 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
         To lessen the effect of regularization on synthetic feature weight
         (and therefore on the intercept) intercept_scaling has to be increased.
 
-    class_weight : dict or 'balanced', optional
+    class_weight : dict or 'balanced', optional, default: None
```

Comment:
@TomDLT I think the convention is to use optional for parameters that represent features that are optional. I think this is okay! @amueller correct?